doc/en/apiref.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
`git mv` does nothing more than moving the file and calling `git rm` and `git add` for you - git basically doesn't know the concept of renaming files.

However, it treats a file as renamed (when showing the log, i.e. that fact is not tracked by git) if there's a removed file and an added file where some (high) percentage of the content is equal - in this case, this detection failed as `.txt` was removed everywhere.